openshift/keycloak.prep.yml

```diff
@@ -34,6 +42,9 @@ objects:
     stringData:
       realm: ${KEYCLOAK_REALM}
       auth-url: ${KEYCLOAK_AUTH_URL}
-      fe-client-id: ${APP_NAME}${KEYCLOAK_FE_CLIENTID_SUFFIX}
+      fe-client-name: ${APP_NAME}${KEYCLOAK_FE_CLIENTID_SUFFIX}
```

Comment:
doesn't this name need to be changed in server.dc.yml as well?

Reply:
Good catch. Fixed.